run-test-tool-scenarios.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -eux
+
+./startup.sh -x clean
```

Comment:
what does the `-x clean` do? (just curious)

Reply:
-x clean is passed as argument to the startup script so that it doesn't clear out the test results xml from Jenkins 1. I think its no longer required I will update it.